### ISSUE TITLE
Fails with missing login configuration

### ISSUE THREAD

**Issue by pwonson** [rundeck-plugins/jira-workflow-step#6]
Project configuration settings:

project.plugin.WorkflowStep.JIRA-Issue-Exists.login=username@rundeck.com
project.plugin.WorkflowStep.JIRA-Issue-Exists.password=keys/jira/jira-demo
project.plugin.WorkflowStep.JIRA-Issue-Exists.url=https\://rundeck.atlassian.net

Error:

Failed: MissingConfiguration: login is required
--
19:30:55 |   | Execution failed: 27825 in project Route53_and_Jira: [Workflow result: , step failures: {6=Dispatch failed on 1 nodes: [Rundeck-1: MissingConfiguration: login is required + {dataContext=MultiDataContextImpl(map={}, base=null)} ]}, Node failures: {Rundeck-1=[MissingConfiguration: login is required + {dataContext=MultiDataContextImpl(map={}, base=null)} ]}, status: failed]

Rundeck Enterprise version: RUNDECK ENTERPRISE 3.2.0-20191218

**Comment by marcjay** [comment]
The plugin doesn't seem to support loading a password from Rundeck's Key Storage - the actual password needs to be entered in plain text into the `project.plugin.WorkflowStep.JIRA-Issue-Exists.password` property

**Comment by pwonson** [comment]
Maybe change to an  enhancement request? All plugins should support getting passwords from key storage.

**Comment by marcjay** [comment]
> Maybe change to an enhancement request? All plugins should support getting passwords from key storage.

Yup I agree - I raised #19 the other day to address this. I'm willing to put a PR together for it if someone confirms this project is still active enough to review and merge it
